frontend/src/actions/messages/messages.js

```diff
@@ -1,9 +1,264 @@
 import { defineMessages } from 'react-intl'
+import { defaultCipherList } from 'constants'
```

Comment:
What is that @flor3s?

Reply:
When i ran `npm i -S react react-intl` it asked me to update dependencies, which I think may have created the added import. I removed `import { defaultCipherList } from 'constants'` from the `messages.js` and was able to run the translation script successfully, so I do not think it is necessary.